mousestyles/visualization/plot_path.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+def plot_path(movement, sep_points, title='example plot of path', alpha=.1,
+    xlim = [-16.24, 3.76], ylim = [0.9, 43.5]):
+    r"""
+    Plot the lines along paths.
+
+    Parameters
+    ----------
+    movement : pandas.DataFrame
+        CX, CY coordinates. Must have length greater than 1.
+
+    sep_points : list
```

Comment:
`sep_points` is the list of path indices from the function `path_index`, right? Maybe we can change the description to make it consistent. 
The following description is what I put in the `filter_path` function.

> paths: list
        a list containing the indices for all paths